src/components/register.js

```diff
@@ -82,7 +82,7 @@ export default class RegisterComponent extends React.Component {
           <p>Connect with parents and education professionals from around the world to make education better for all children in all schools and families worldwide.</p>
         </div>
       </header>
-      <form action="" onSubmit={this.submitHandler} className="layout__row">
+      <form action="" onSubmit={this.submitHandler} onClick={this.props.onClick} className="layout__row">
```

Comment:
`form.onClick` is a handler for auth page where the click on register form needs to be caught. But maybe I should use `div` wrapper for reg form there instead of giving `this.props.onClick` to `RegisterComponent`. Yes?